src/components/QuoteApi.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+
+function Quote() {
+  const [quotes, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorPresent, setError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      setIsLoading(true);
+      try {
+        const response = await fetch('https://api.api-ninjas.com/v1/quotes/', {
+          method: 'GET',
+          headers: {
+            'X-Api-Key': 'SxVyw4SCwQ8g6rx2kEDbkA==vs1eTXQwIMNugbr1',
+          },
+        });
+        const data = await response.json();
+        setQuote(data);
+      } catch (error) {
+        console.error('Error fetching data', error);
+        setError(true);
+      }
+      setIsLoading(false);
+    };
+
+    fetchQuote();
+
+    // Clean up when component unmounts
+    return () => {
+      console.log('Quote Clean up!');
+    };
```

Comment:
- [ ] Please do not commit `console.log` to your repo. It kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.